install/kubernetes/istio-auth.yaml

```diff
@@ -1238,6 +1238,16 @@ spec:
             fieldRef:
               apiVersion: v1
               fieldPath: metadata.namespace
+        # While using this file to deploy istio, due to INSTANCE_IP net set, the xDS key
+        # to query pilot doesn't have a valid ip address. Therefore, HostInstances() call
+        # returns nothing, and therefore is not needed for the ingress. Is that a bug?
```

Comment:
It's deliberate, since we shouldn't be calling `HostInstances` from the ingress proxy. It should be only sending ingress routes which do not depend on the ingress proxy IP.